Makefile

```diff
@@ -8,53 +10,66 @@ help:
 build: ## Build project
 	dotnet build -f=netcoreapp2.0
 
-.PHONY: test
```

Comment:
I think we still want to keep `test` as this keeps a common interface with all our other clients. We use this interface in concourse - https://github.com/alphagov/notifications-aws/blob/master/concourse/client-library-pipeline.yml.j2#L193